### PR TITLE
update silly wisher assets to 0.35

### PR DESCRIPTION
**Pull request by furrycatherder**
## Describe your changes

Update the Silly Wisher assets to 0.35.

## Issue or discord link

- 

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

**Review by nguyentvan7 — COMMENTED** [review]
image
xianyun looks a lot like gaming

**Review by nguyentvan7 — APPROVED** [review]

**Comment by furrycatherder** [comment]
Should be good to go now. 🙂

**Review by nguyentvan7 — DISMISSED** [review]

File: libs/gi/silly-wisher/src/portraits/index.ts [hunk 1]
```diff
@@ -67,6 +68,7 @@ import Venti from './portrait_venti.png'
 import Wanderer from './portrait_wanderer.png'
 import Wriothesley from './portrait_wriothesley.png'
 import Xiangling from './portrait_xiangling.png'
+import Xianyun from './portrait_xiangyun.png'
```
Comment (frzyc) [inline libs/gi/silly-wisher/src/portraits/index.ts:71]:
```suggestion
import Xianyun from './portrait_xianyun.png'
```

File: libs/gi/silly-wisher/src/splash/index.ts [hunk 2]
```diff
@@ -67,6 +68,7 @@ import Venti from './splash_venti.png'
 import Wanderer from './splash_wanderer.png'
 import Wriothesley from './splash_wriothesley.png'
 import Xiangling from './splash_xiangling.png'
+import Xianyun from './splash_xiangyun.png'
```
Comment (frzyc) [inline libs/gi/silly-wisher/src/splash/index.ts:71]:
```suggestion
import Xianyun from './splash_xianyun.png'
```

**Review by frzyc — CHANGES_REQUESTED** [review]
Looks like `Xianyun` is mispelled, not sure why this was not being picked up from pipeline.

**Review by frzyc — APPROVED** [review]

**Pull request merged**